x/gov/types/msgs.go

```diff
@@ -18,6 +18,12 @@ const (
 	TypeMsgVote           = "vote"
 	TypeMsgVoteWeighted   = "weighted_vote"
 	TypeMsgSubmitProposal = "submit_proposal"
+
+	// These are used for querying events by action.
+	TypeSvcMsgDeposit        = "/cosmos.gov.v1beta1.Msg/Deposit"
```

Comment:
Same comment as bellow - are these FQ paths not defined elsehwere already? It feels like this is will be a very easy thing to forget to update when we migrate out of beta, change packages, etc. Yes this shouldn't happen often... but when it does, I worry these const strings may get left behind..

Reply:
Unfortunately not... In Aaron's custom code generator they are, and i think that's useful.